acceptance/framework/vault/vault_cluster.go

```diff
@@ -29,6 +29,7 @@ import (
 const (
 	releaseLabel                 = "app.kubernetes.io/instance="
 	defaultVaultHelmChartVersion = "v0.21.0"
+	//defaultVaultHelmChartVersion = vaultVersion
```

Comment:
Please consider removing this.. it is generally better to not commit commented out code.

Ignore me if you're still working on this bit.